### PR TITLE
WIP: Add sigreturn command

### PR DESCRIPTION
**Pull request by gsingh93**
A command to dump the `ucontext_t` at the top of the stack, useful for debugging SROP exploits. Still a WIP, but what I'm most confused about is what to call this command. `sigreturn`, `srop`, `ucontext`, or something else?

### CODE REVIEW

**Comment by gsingh93** [comment]
@disconnect3d what do you think this command should be called? Once I know that I think this will be easy to finish up.

**Comment by disconnect3d** [comment]
@gsingh93 srop or sigreturn

File: pwndbg/commands/sigreturn.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import pwndbg.commands
+import pwndbg.gdblib.arch
+import pwndbg.gdblib.memory
+import pwndbg.gdblib.regs
+
+parser = argparse.ArgumentParser(description="")
+
+
+@pwndbg.commands.ArgparsedCommand(parser)
+def sigreturn():
```
Comment (disconnect3d) [inline pwndbg/commands/sigreturn.py:12]: only x86-64 or sth

File: pwndbg/commands/sigreturn.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "rbx",
+            "rdx",
+            "rax",
+            "rcx",
+            "rsp",
+            "rip",
+            "eflags",
+        ]
+    ):
+        offset = 0x28 + i * 8
```
Comment (disconnect3d) [inline pwndbg/commands/sigreturn.py:36]: arch ptrsize?

**Comment by gsingh93** [comment]
Was implemented in https://github.com/pwndbg/pwndbg/pull/1940, closing.

**Pull request closed without merging**